interface-definitions/protocols-mpls.xml.in

```diff
@@ -26,6 +26,51 @@
                   </constraint>
                 </properties>
               </leafNode>
+              <node name="allocation">
+                <properties>
+                  <help>Forwarding equivalence class (FEC) allocation from local routes</help>
+                </properties>
+                <children>
+                  <node name="ipv4">
+                    <properties>
+                      <help>IPv4 routes</help>
+                    </properties>
+                    <children>
+                      <leafNode name="access-list">
+                        <properties>
+                          <help>Access-list number</help>
+                          <valueHelp>
+                            <format>1-2699</format>
```

Comment:
Why is this limited to only 2699 access-lists? Is this a read FRR limit or is it an artificial one?